### PR TITLE
bruteforce: Add export button

### PR DESCRIPTION
**Pull request by kingthorin**
- Add a TableExportButton
- Tweak help (mention the button, and tweak an external link).

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/bruteforce/bruteforce.gradle.kts [hunk 1]
```diff
@@ -10,7 +10,7 @@ tasks.withType<JavaCompile> {
 zapAddOn {
     addOnName.set("Forced Browse")
     addOnStatus.set(AddOnStatus.BETA)
-    zapVersion.set("2.7.0")
+    zapVersion.set("2.8.0")
```
Comment (thc202) [inline addOns/bruteforce/bruteforce.gradle.kts]: Worth mentioning in the changelog.

File: addOns/bruteforce/src/main/javahelp/org/zaproxy/zap/extension/bruteforce/resources/help/contents/concepts.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <H1>Forced Browse</H1>
 	<p>
 		Forced Browse is configured using the <a href="options.html">Options
 			Forced Browse screen</a>.<br />
-	<p>This functionality is based on code from the OWASP DirBuster project.
+	<p>This functionality is based on code from the <a href="http://www.owasp.org/index.php/Category:OWASP_DirBuster_Project">OWASP DirBuster project</a>.
```
Comment (thc202) [inline addOns/bruteforce/src/main/javahelp/org/zaproxy/zap/extension/bruteforce/resources/help/contents/concepts.html]: Change to https?

**Comment by kingthorin** [comment]
Tweaked.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**